python/src/BernsteinCopulaFactory_doc.i.in

```diff
@@ -1,9 +1,23 @@
 %feature("docstring") OT::BernsteinCopulaFactory
-"BernsteinCopula copula factory.
+"EmpiricalBernsteinCopula factory.
 
 Notes
 -----
-This class provides a non parametric estimator of a copula: the :class:`~openturns.EmpiricalBernsteinCopula`.
+This class builds an :class:`~openturns.EmpiricalBernsteinCopula` which is a non parametric fitting of
+the copula of a multivariate distribution.
+
+The keys of :class:`~openturns.ResourceMap` related to the class are:
+
+- the keys `BernsteinCopulaFactory-MinM` and `BernsteinCopulaFactory-MaxM` that define the default range of :math:`m` in the optimization
+  problems computing the optimal bin number according to a specified criteria,
+- the key `BernsteinCopulaFactory-BinNumberSelection` that defines the default criteria to compute the optimal bin number when it is not
```

Comment:
The possible choices are 'AMISE', 'LogLikelihood', 'PenalizedCsiszarDivergence', the latest being never used due to the awful spelling of the key!